.changeset/sour-shrimps-start.md

```diff
@@ -2,7 +2,7 @@
 "@fluid-internal/client-utils": minor
 ---
 
-Internal buffer encoding helpers now require 'utf8', 'utf-8', or 'base64'
+client-utils: Internal buffer encoding helpers now require 'utf8', 'utf-8', or 'base64'
```

Comment:
Curious: did you manually remove this one from the release notes file? I agree it's not relevant there, this being an internal package, just wondering how come it didn't show up.

Reply:
I think this was a copy/paste fail when I reordered the markdown.

Reply:
But not going to worry about fixing it since as you noted it's an internal package. The note will still make it into the client-utils changelog.